apps/rod/views/reuse-main-applicant-address.html

```diff
@@ -0,0 +1,21 @@
+{{<partials-page}}
+  {{$page-content}}
+  {{#markdown}}delivery-address-for-documents{{/markdown}} 
+
+  <div class="govuk-details__text">
+  <div class="govuk-summary-list__row summary-list-box" >
+        <div>{{values.main-applicant-address-1}}</div>
+        <div>{{values.main-applicant-address-2}}</div>
```

Comment:
Perhaps better to render this conditionally so we don't end up rendering an empty div when address line is 2 is not entered.